src/store/modules/sales.js

```diff
@@ -33,6 +34,9 @@ export default {
     async postSale({ commit }, data) {
       await postSale(data);
     },
+    async returnSale({ commit }, productId) {
```

Comment:
same with this. We are returning a product

Reply:
fixed